forem/forem
# Add "dashboard" icon to member_manager view for Admins

## What type of PR is this? (check all applicable)

- [x ] Optimization

## Description
Adds Dashboard icon along with link to user dashboard

## Related Tickets & Documents



- Related Issue #
- Closes #19087 

## QA Instructions, Screenshots, Recordings
 - Proceed to `/admin/member_manager`
 - Click on any user.
 - Click the dashboard icon and it should take you to `/dashboard/:username` where `username` is the user's username.

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/lefddVT71wmMoH0AMN/giphy.gif)
